packages/client/src/index.test.ts

```diff
@@ -58,29 +59,26 @@ const generateRandomString = (length: number) =>
 const access_token = generateRandomString(43);
 const refresh_token = generateRandomString(43);
 const sub = generateRandomString(8);
-const id_token = jwt.sign(
-  {
-    sub,
-    at_hash: 'RHPz55byGq-p81hfzGVYfA',
-    aud: 'foo',
-    exp: Math.floor(Date.now() / 1000) + 1000,
-    iat: Math.floor(Date.now() / 1000),
-    iss: 'https://logto.dev/oidc',
-  },
-  'secret'
-);
 const scope = 'openid offline_access';
 const token_type = 'Bearer';
 
 describe('setToken', () => {
   let client: LogtoClient;
+  let id_token: string;
   beforeAll((done) => {
     client = new LogtoClient(
       {
         logtoUrl: 'https://logto.dev',
         clientId: 'foo',
       },
-      () => {
+      async () => {
+        id_token = await new SignJWT({})
+          .setProtectedHeader({ alg: 'HS256' })
```

Comment:
should we use RS256 sign algorithm by default?

Reply:
fixed, although it doesn't matter in this case, but rs256 would be better.